### DISCUSSION TITLE
Representing vulnerabilities

### DISCUSSION THREAD

**Discussion by psiinon** [ossf/wg-security-tooling#5] [General]
On the last call I mentioned that some OWAS Pfolk were looking at a way of representing DAST vulnerabilities - this is for deliberately vulnerable apps to document which vulnerabilities they expose.
Right now theres an active discussion on https://github.com/bkimminich/juice-shop/issues/1441 so please chip in if you have any opinions on this.

**Comment by greysteil** [comment]
Cc @sj
